src/lib.rs

```diff
@@ -580,6 +580,9 @@ pub fn encode_many_config(pems: &[Pem], config: EncodeConfig) -> String {
         .join(line_ending)
 }
 
+#[cfg(feature = "serde")]
+extern crate serde_core as serde;
+
```

Comment:
I would _rather_ not have this here and instead have the references to `serde` in the `serde_impl` module changed to be `serde_core`. I get that it is a bigger change, but it is cleaner that way.